### ISSUE TITLE
Move r4ss into `nmfs-stock-synthesis` organization?

### ISSUE THREAD

**Issue by iantaylor-NOAA** [r4ss/r4ss#815]
Moving the r4ss repo (and potentially ss3sim) into the `nmfs-stock-synthesis` organization (which didn't exist when the r4ss organization on github was created) could consolidate some of the overhead associated with having these related repositories in separate places.

Checklist below copied from https://github.com/nmfs-stock-synthesis/workflows/issues/30.
- [x] move over the r4ss/shiny-selex repo 
- [ ] Learn from this experience to decide the details of how to migrate r4ss
- [ ] Clean up other r4ss org repos that aren't used (e.g., r4ss::SS_utils can probably just be deleted; likely true for other repos)
- [ ] Move r4ss over to nmfs-stock-synthesis
- [ ] Announce move of r4ss in release notes
- [ ] Decide on strategy for redirecting folks to the new location of r4ss (e.g., looking at apps that depend on r4ss and letting them know about the change? Potentially forking the new location of r4ss back to the old and adding a message upon trying to download or use r4ss from the old location?

**Comment by e-perl-NOAA** [comment]
This would now be moving it over to enterprise on `nmfs-ost`. @iantaylor-NOAA just let me know when you want to move it over and we can. We would just want to create some redirects first.

**Comment by iantaylor-NOAA** [comment]
@MOshima-PIFSC and @efletcherPIFSC, I see that you have a plan to move ss3diags to nmfs-ost. Maybe we should coordinate and move r4ss at the same time (which has been the subject of this long-standing but newly renamed issue)? Do you have a timeline in mind?

**Comment by MOshima-PIFSC** [comment]
@iantaylor-NOAA yes it would probably be best to coordinate on this and move ss3diags and r4ss at the same time. We need to do a few things to clean up the code base before we move it and realistically, we were thinking if we can transition it before the end of the year that would be good timing. Would that work for you? One thing we were discussing is if we should update our code to work with the newest release of r4ss (regarding variable re-naming). My thinking is we will need to do it eventually, so it would probably be better to do it before we transition that way we can include it in the announcement. But I don't think that would take much time, so I think end of the year would still be feasible.

**Comment by iantaylor-NOAA** [comment]
End of the year sounds good to me. I would probably target either before Monday Dec 16, or after Monday Jan 6 to reduce conflict with holidays and ensure that users are around to test the new location and we're around to support them if there are any issues.

**Comment by MOshima-PIFSC** [comment]
That's a good point. I would prefer to shoot for before Dec 16th. January will be a busy month for me with meetings and travel. But if January works better for others, that is fine with me.

**Comment by efletcherPIFSC** [comment]
In the short term, ss3diags can be patched w/ r4ss 1.49.3 dependency. Then, the next major version will have r4ss 1.50.0 compat in mind (PIFSCstockassessments/ss3diags#109)

**Comment by iantaylor-NOAA** [comment]
@MOshima-PIFSC and @efletcherPIFSC, I just had a discussion with @e-perl-NOAA about this move and we both think it might be better to wait until the issues around SAML, access tokens etc. for github enterprise are better resolved before moving these repos to nmfs-ost. At this point it doesn't feel like there's a rush to get things moved.

What do you think?

**Comment by MOshima-PIFSC** [comment]
@iantaylor-NOAA That's a good idea, makes sense to me. Hopefully this issue is resolved!

**Comment by kellijohnson-NOAA** [comment]
In my experience pushing PRs to tidyverse packages is that they do eventually get in if the Issue is not rejected right off of the bat but they leave PR open for a long time. It appears as though they have a rotating schedule with respect to which packages they are working on and they will merge multiple PRs in at one time rather than dealing with each PR as they come in.
